WordPress/gutenberg
# Use PostCSS + PostCSS plugins for style transformation

## What?
This PR uses PostCSS and PostCSS plugins for style transformations.

## Why?
The internal CSS parser has compatibility and stability issues, as it is not a dedicated, separate library.

## How?
Using PostCSS ensures that a dedicated and well-supported library is used.

## Testing Instructions
1. Use the Gutenberg plugin with this PR (branch) applied.
2. Add frontend styles or other styles as editor stylesheet that are known to cause issues with the custom CSS parser (e.g. https://github.com/WordPress/gutenberg/issues/21569; https://github.com/WordPress/gutenberg/issues/19930 and from other issues).
3. Open the Gutenberg editor, 
note that the editor styles are correctly post-processed and applied, resulting in correctly wrapped selectors and rewritten URLs.
4. Add the HTML Preview block and check that adding custom styles do not break it.


Related PR: https://github.com/WordPress/gutenberg/pull/49483
